dep/reflex-platform/github.json

```diff
@@ -1,7 +1,7 @@
 {
   "owner": "reflex-frp",
   "repo": "reflex-platform",
-  "branch": "develop",
-  "rev": "90351bdb2c99d416ea95fd5b8b474e8f9fab2942",
-  "sha256": "0bbzz4ckdx0ilhhpd3drfbsc0zdb2rnhw2m3nls9villbs3lz3lh"
+  "branch": "aa-reflex-ghci",
```

Comment:
I'll at least wait till this gets in.

Reply:
Seems to be merged https://github.com/reflex-frp/reflex-platform/pull/620/commits

Reply:
Awesome. Just need to update the branch here then.